akka-sample-distributed-workers-scala/src/main/scala/worker/Master.scala

```diff
@@ -0,0 +1,185 @@
+package worker
+
+import java.util
+
+import akka.actor.typed.Behavior
+import akka.actor.typed.scaladsl.Behaviors
+import akka.actor.typed.ActorRef
+import akka.actor.typed.receptionist.{ Receptionist, ServiceKey }
+import akka.cluster.pubsub.{ DistributedPubSub, DistributedPubSubMediator }
+import akka.persistence.typed.PersistenceId
+import akka.persistence.typed.scaladsl.Effect
+import akka.persistence.typed.scaladsl.EventSourcedBehavior
+import worker.WorkState.WorkAccepted
+import worker.WorkState.WorkDomainEvent
+import akka.actor.typed.scaladsl.adapter._
+import worker.WorkState.WorkCompleted
+import worker.WorkState.WorkStarted
+import worker.WorkState.WorkerFailed
+
+import scala.collection.mutable.ListBuffer
+import scala.concurrent.duration.{ Deadline, FiniteDuration, _ }
+
+/**
+ * The master actor keep tracks of all available workers, and all scheduled and ongoing work items
+ */
+object Master {
```

Comment:
Should we call it something else than master (not sure it is worth it but it does makes me think of master-slave) and it really is more of a `WorkManager` or something anyways or what is it the master of?

Reply:
Yeah best to remove that kind of terminology